### PR TITLE
[FlowBundle] Fix version so we can use this bundle with Symfony 2.8 too

### PR DESCRIPTION
**Pull request by tristanbes**

### CODE REVIEW

File: src/Sylius/Bundle/FlowBundle/composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": ">=5.5.9",
 
-        "symfony/framework-bundle": "~2.7.5"
+        "symfony/framework-bundle": "~2.7,>=2.7.5"
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/FlowBundle/composer.json]: We can simply use `^` instead of `~` and that should be corrected in all bundles...

Comment (pamil) [inline src/Sylius/Bundle/FlowBundle/composer.json]: Should be `^2.7.7`, versions from `2.7` to `2.7.6` have [some security issues](http://symfony.com/blog/symfony-2-7-7-released).

**Comment by tristanbes** [comment]
done

**Comment by pjedrzejewski** [comment]
Thank you Tristan!

**Pull request merged**